bedrock/mozorg/templates/mozorg/about/policy/transparency/index.html

```diff
@@ -0,0 +1,252 @@
+              <p>
+                {{ _('Users are notified if we receive a <a href="%s">Takedown Notice</a> related to their submission on a Mozilla service.')|format('#dfn-takedown-notice') }}
+                {{ _('We also try to publicly post copies of the Takedown Notices (with personal data redacted) to sites such as <a href="%s">MozWiki</a> and <a href="%s">Lumen Database</a> (formerly known as the Chilling Effects project).')|format('https://wiki.mozilla.org/Legal/Infringement_Notices','https://lumendatabase.org/') }}
+            </div>
+          </section>
+
+          <section>
+            <h3 data-accordion-role="tab">{{ _('What does the Supplement cover?') }}</h3>
+            <div data-accordion-role="tabpanel">
+              <p>
+                {{ _('This section of our report covers situations that don’t fit into our reporting categories.') }}
+                {{ _('For example, to the extent we are legally permitted, we may include voluntary disclosures as well as legal and policy activities that we engaged in during the reporting period to further government transparency.') }}
+              </p>
+            </div>
+          </section>
+        </div>
+
+      </div>
+    </section>
+
+    <section class="section section-extra section-defs" id="definitions">
+      <div class="content">
+        <h2 class="section-title">{{ _('Definitions') }}</h2>
+
+        <dl class="simple">
+          <dt id="dfn-emergency-request">{{ _('Emergency Request') }}</dt>
+          <dd>
+            <p>{{ _('A request from a government agency seeking information on an expedited basis in connection with an emergency, typically involving death or serious injury.') }}</p>
+          </dd>
+
+          <dt id="dfn-court-order">{{ _('Court Order') }}</dt>
+          <dd>
+            <p>{{ _('An order issued by a judge or magistrate compelling a company to engage or refrain from certain action.') }}</p>
+          </dd>
+
+          <dt id="dfn-counter-notice">{{ _('Counter Notice') }}</dt>
+          <dd>
+            <p>{{ _('Documentation that meets the counter notification requirements <a href="%s">set forth here</a> in response to a <a href="%s">Takedown Notice</a>.')|format(url('legal.report-infringement'), url('mozorg.about.policy.transparency.index') + '#dfn-takedown-notice') }}</p>
+          </dd>
+
+          <dt id="dfn-threat-indicator">{{ _('Cybersecurity Threat Indicator') }}</dt>
+          <dd>
+            <p>{{ _('Pieces of information about a threat to a computer network or system, such as a vulnerability, piece of malicious code, or the IP address of an attacker. This definition is based on the Cybersecurity Information Sharing Act of 2015 (CISA); the full definition is at <a href="%s">6 U.S.C. § 1501(6)</a>.')|format('http://uscode.house.gov/view.xhtml?req=%28title:6%20section:1501%20edition:prelim%29%20OR%20%28granuleid:USC-prelim-title6-section1501%29&amp;f=treesort&amp;edition=prelim&amp;num=0&amp;jumpTo=true') }}</p>
```

Comment:
Looks like the `&amp;`s are at fault. It's a clunky mix of encoding. I'll clean it up.